src/energyplus/Test/ScheduleInterval_GTest.cpp

```diff
@@ -277,6 +277,113 @@ TEST_F(EnergyPlusFixture, ForwardTranslator_ScheduleFixedInterval_Hourly_Shifted
   EXPECT_EQ(8760, numUntils);
 }
 
+TEST_F(EnergyPlusFixture, ForwardTranslator_ScheduleFixedInterval_Days) {
+  // timestep 1 day
+  {
+    Model model;
+    Date start_date(MonthOfYear::Jan, 1, 2007);
+    Time timestep_day(1, 0);
+    Vector daily_wh_inlet_temperatures_c(5);
+    daily_wh_inlet_temperatures_c[0] = 10.0422222222222;
+    daily_wh_inlet_temperatures_c[1] = 9.98111111111111;
+    daily_wh_inlet_temperatures_c[2] = 9.92111111111111;
+    daily_wh_inlet_temperatures_c[3] = 9.86222222222222;
+    daily_wh_inlet_temperatures_c[4] = 9.80444444444445;
+    TimeSeries time_series_tmains(start_date, timestep_day, daily_wh_inlet_temperatures_c, "C");
+
+    boost::optional<ScheduleInterval> scheduleInterval = ScheduleInterval::fromTimeSeries(time_series_tmains, model);
+    ASSERT_TRUE(scheduleInterval);
+    EXPECT_TRUE(scheduleInterval->optionalCast<ScheduleFixedInterval>());
+    ScheduleFixedInterval scheduleFixedInterval = scheduleInterval->optionalCast<ScheduleFixedInterval>().get();
+    EXPECT_EQ(1440, scheduleFixedInterval.intervalLength());  // one day in minutes
+    EXPECT_EQ(1, scheduleFixedInterval.startMonth());
+    EXPECT_EQ(2, scheduleFixedInterval.startDay());  // unrelated to this test, but this should be fixed to be 1
```

Comment:
That feels weird no? Should we address that?

Reply:
I've already created [this issue](https://github.com/NREL/OpenStudio/issues/4707) for addressing that.